src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -15,6 +15,11 @@
 
 namespace Microsoft.ML.Runtime.FastTree
 {
+    [TlcModule.ComponentKind("FastTreeBinaryTrainer")]
```

Comment:
>ComponentKind [](start = 15, length = 13)

Are you sure you need this as a component?..

Reply:
What should it be instead?

---
In reply to: [208784667](https://github.com/dotnet/machinelearning/pull/653#discussion_r208784667) [](ancestors = 208784667)